openssl_test.go

```diff
@@ -58,7 +58,7 @@ func TestMain(m *testing.M) {
 	fmt.Println("OpenSSL version:", openssl.VersionText())
 	fmt.Println("FIPS enabled:", openssl.FIPS())
 	status := m.Run()
-	for i := 0; i < 5; i++ {
+	for _ = range 5 {
```

Comment:
Is the assignment necessary?

```suggestion
	for range 5 {
```